vector/src/main/java/im/vector/app/features/home/room/detail/timeline/helper/MatrixItemColorProvider.kt

```diff
@@ -107,12 +105,28 @@ class MatrixItemColorProvider @Inject constructor(
             }
         }
 
-        @ColorRes
+        
+        @ColorInt
+        private fun getUserColorByIndexColorInt(colorProvider: ColorProvider, index: Int): Int {
```

Comment:
I think the fun above `getUserColorByIndex` could be removed.
And so the resources `R.color.element_name_0x` and `R.color.element_room_0x`.